app/models/wakes/location.rb

```diff
@@ -13,7 +14,8 @@ def label
     path
   end
 
-  def url(protocol: 'http', host: ENV['DEFAULT_HOST'])
+  def url(protocol: 'http', host: nil)
+    host ||= self.host || ENV['DEFAULT_HOST']
```

Comment:
@benhutton good points! 

@chrismaddocks @benhutton 
I am an favor of using `host = override_host || self.host || ENV['DEFAULT_HOST']` as a next step, if not what Ben suggests in #2. I prefer calling it `host_override` instead of `override_host`.